smcontroller/smcontroller.go

```diff
@@ -116,6 +132,9 @@ func New(
 		systemQuotaAlertChan:      make(chan cloudprotocol.SystemQuotaAlert, statusChanSize),
 		nodeConfigStatusChan:      make(chan unitconfig.NodeConfigStatus, statusChanSize),
 		nodes:                     make(map[string]*smHandler),
+		certChan:                  make(<-chan *iamanager.CertInfo),
```

Comment:
Make is useless here.

Reply:
make initializes certChannel  for server.handleChannels() goroutine for insecure connection case